tools/codegen/gen_lazy_tensor.py

```diff
@@ -200,6 +200,7 @@ def concat_map_codegen(func: Callable[[NativeFunction], Sequence[str]],
                 "torch/csrc/lazy/core/ir.h",
                 "lazy_tensor_core/csrc/ts_backend/ts_node_lowering.h",
                 "torch/csrc/lazy/core/hash.h",
+                "lazy_tensors/str_join.h",
```

Comment:
Is this easy to avoid?  @alanwaketan was replacing all of these with c10::Str(a, b) i think.

Reply:
StrJoin was already gone. What's left in str_join.h is a bunch of ToString methods, which are mostly only used in the generated LazyLazyIr.h.

I will do the clean up in another PR. Maybe we could just codegen those methods in LazyLazyIr.h